src/components/claculator.js

```diff
@@ -0,0 +1,27 @@
+export default function Myname() {
+  return (
```

Comment:
- [ ] Good work here. I suggest you consider changing the component name to `Calculator`, to give the component a better description that fits the functionality 👍 
- Also, you spelled the file name wrongly. its meant to be `Calculator.js` not "claculator.js". Kindly make this change as it will show more attention to detail and adherence is maintainable code